### PR TITLE
Remove expired RFP

### PR DESCRIPTION
**Pull request by g4s8**
#574 - added stakeholder to remove expired RFPs

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#828](https://github.com/zerocracy/farm/issues/828) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#828](https://github.com/zerocracy/farm/issues/828) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/751531200). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/main/resources/com/zerocracy/stk/pmo/rfps/remove_expired_rfps.groovy [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import com.zerocracy.pmo.Rfps
+
+def exec(Project pmo, XML xml) {
+  new Assume(pmo, xml).isPmo()
+  new Assume(pmo, xml).type('Ping daily')
+  ClaimIn claim = new ClaimIn(xml)
+  Date expiration = claim.created() - new Policy().get('41.days', 32)
+  Rfps rfps = new Rfps(pmo).bootstrap()
+  rfps.olderThan(expiration).each { rfp ->
+    rfps.remove(rfp)
```
Comment (carlosmiranda) [inline src/main/resources/com/zerocracy/stk/pmo/rfps/remove_expired_rfps.groovy]: @g4s8 Perhaps we should make a new claim "RFP was removed" here, for possible future reference.

**Comment by g4s8** [comment]
@carlosmiranda right, fixed 6cf1691

**Comment by carlosmiranda** [comment]
@yegor256 good to merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by g4s8** [comment]
@yegor256 Rultor's build failed on `com.zerocracy.tools.SignatureTest`,  I don't think this test related to PR somehow, could you please restart merge? I tried to build locally and it's ok.

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#828](https://github.com/zerocracy/farm/issues/828) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 points just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**